src/main/java/blang/distributions/BetaBinomial.bl

```diff
@@ -0,0 +1,21 @@
+package blang.distributions
+
+model BetaBinomial{
+	random IntVar realization
+	param IntVar numberOfTrials
+	param RealVar alpha
+	param RealVar beta
+	
+	laws{
+		logf(realization,numberOfTrials,alpha,beta)
+		{
+			if(realization<0) return NEGATIVE_INFINITY
+			if(numberOfTrials<=0 || realization> numberOfTrials) return NEGATIVE_INFINITY
+		return lnGamma(numberOfTrials+1)+lnGamma(realization+alpha)+lnGamma(numberOfTrials-realization+beta)+lnGamma(alpha+beta)
```

Comment:
Please fix indentation. Also avoid using tabs, instead rely on spaces (ideally, pick one tab == 2 spaces). Space between if and opening parenthesis too.